apps/web/test/lib/getSchedule.test.ts

```diff
@@ -645,9 +645,8 @@ describe("getSchedule", () => {
       expect(scheduleForEventWithBookingNotice13Hrs).toHaveTimeSlots(
         [
           /*`04:00:00.000Z`, `06:00:00.000Z`, - Minimum time slot is 07:30 UTC which is 13hrs from 18:30*/
```

Comment:
I guess the slots are coming now as expected here in comments (or manual calculation).
`Minimum time slot is 07:30 UTC....`